docs/core/tools/dotnet-new.md

```diff
@@ -155,6 +167,34 @@ The command calls the [template engine](https://github.com/dotnet/templating) to
 
   Checks if there are updates available for the template packs that are currently installed. Available since .NET Core 3.0 SDK.
 
+- **`--search`**
```

Comment:
@KathleenDollard could you please check the order of commands and let me know if it should be adapted?

Reply:
We generally try to put the options in alphabetical order.

Reply:
To add to this. I think if we just do this long list of marginally related items, it will not be a good experience for the reader. So, let's look for some alternatives. Grouping under the dependent options seems the nicest to me.

Reply:
We could create a separate H2 section named "Search and list options" following the "Options" H2 section.